test/spec/modules/eightPodAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,180 @@
+import analyticsAdapter, { storage, queue, context, trackEvent } from 'modules/eightPodAnalyticsAdapter.js';
+import { expect } from 'chai';
+import adapterManager from 'src/adapterManager.js';
+import eightPodAnalytics from 'modules/eightPodAnalyticsAdapter.js';
+import CONSTANTS from '../../../src/constants.json';
+
+const {
+  EVENTS: {
+    BID_WON
+  }
+} = CONSTANTS;
+
+describe('', function() {
```

Comment:
Please identify this as the adapter, it will make these tests much easier to identify if there are failures.